app/scripts/services/httpUrlInterceptor.js

```diff
@@ -3,7 +3,11 @@ angular
   .factory('httpUrlInterceptor', function (envService) {
     return {
       request: function (config) {
-        var passthroughRegexs = [/https?:\/\/.*/, /^views\/.*/, /template\/.*/];
+        var passthroughRegexs = [
+          /https?:\/\/.*/,
+          /^\/views\/.*/,
```

Comment:
What was the cause for this update?

Reply:
The URL used to load the view HTML files changed from `views/...` to `/views/...`. Probably because of an update to `ngtemplate-loader`.